source/elements/oneMKL/source/domains/blas/omatadd_batch.rst

```diff
@@ -0,0 +1,517 @@
+.. SPDX-FileCopyrightText: 2022 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_blas_omatadd_batch:
+
+omatadd_batch
+=============
+
+Computes a group of out-of-place scaled matrix additions using general
+matrices.
+
+.. _onemkl_blas_omatadd_batch_description:
+
+.. rubric:: Description
+
+The ``omatadd_batch`` routines perform a series of out-of-place scaled matrix
+additions. They are batched versions of :ref:`onemkl_blas_omatadd`,
+but the ``omatadd_batch`` routines perform their operations with
+groups of matrices. Each group contains matrices with the same parameters.
+
+The matrices are always in a strided format for this API.
+
+``omatadd_batch`` supports the following precisions:
+
+   .. list-table::
+      :header-rows: 1
+
+      * -  T 
+      * -  ``float`` 
+      * -  ``double`` 
+      * -  ``std::complex<float>`` 
+      * -  ``std::complex<double>`` 
+
+.. _onemkl_blas_omatadd_batch_buffer:
+
+omatadd_batch (Buffer Version)
+------------------------------
+
+.. rubric:: Description
+
```

Comment:
Please add a note here like omatadd_batch about which APIs this buffer version supports.